### PR TITLE
Added strings.xml in Arabic

### LINKED ISSUE #2037: Adding language - Arabic
**Actual Behaviour**
App only has two languages

**Expected Behaviour**
Adding Arabic translation for strings.xml

**Steps to reproduce it**

strings.xml in Arabic

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by IzhanAli**
Fixes #2037 

**Changes**: Translated strings.xml into Arabic Language

**Screenshot/s for the changes**: [Add screenshot/s of the layout where you made changes or a `*.gif` containing a demonstration]

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: It doesn't need

### CODE REVIEW

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -0,0 +1,981 @@
...
+    <string name="device_not_found">لم يتم العثور على جهاز USB</string>
+    <string name="sensor_not_connected_tls">جهاز استشعار غير متصل</string>
+
+    <string name="time_unit_la">الوقت (مللي ثانية)</string>
+    <string name="view">رأي</string>
+
+
+
+
+
```
Comment (nikit19) [inline app/src/main/res/values-ar/strings.xml]: Please remove these blank lines

Comment (IzhanAli) [reply app/src/main/res/values-ar/strings.xml]: Done!

**Comment by IzhanAli** [comment]
@nikit19 Please review and merge

**Comment by IzhanAli** [comment]
@adityastic Please review this :)

**Comment by adityastic** [comment]
@CloudyPadmal please review

**Comment by IzhanAli** [comment]
Just 24 hrs left for the task :( @CloudyPadmal @adityastic @nikit19

**Comment by adityastic** [comment]
@IzhanAli don't worry I'll extend it for you

**Comment by IzhanAli** [comment]
@adiyastic Thanks

**Comment by IzhanAli** [comment]
It's taking longer than expected :(
Can someone please tell how much time will it take to review and approve changes?

**Comment by adityastic** [comment]
@IzhanAli It's the end of year, people usually arnt working at this time of the year. Please be patient and wait till 5th Jan

**Comment by cweitat** [comment]
do you have a screenshot or GIF to show how it looks?

**Pull request merged**